### PR TITLE
Added Majji.java

### PR DESCRIPTION
**Pull request by PraveenMajji**
I have added one new package and one class in that please go through it

### CODE REVIEW

File: Automation_basics/src/PraveenMajji/Majji.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package PraveenMajji;
+
```
Comment (Pragathishreela) [inline Automation_basics/src/PraveenMajji/Majji.java]: please mention autor name

Comment (PraveenMajji) [reply Automation_basics/src/PraveenMajji/Majji.java]: ok ill add it sorry for my mistake

**Comment by PraveenMajji** [comment]
as you said i have added author name

**Comment by PraveenMajji** [comment]
please merge into your repo

**Pull request merged**